# HG changeset patch
# User Rafael Laboissiere <rafael@debian.org>
# Date 1205863602 14400
#      Tue Mar 18 14:06:42 2008 -0400
# Branch release-3-0-x
# Node ID 65e0f7b770c9f67ac8566054697f1709758f5aad
# Parent  28d36331b049d0b2e7912c40f64f6a43114767f7
Escape "-" signs that mean the "minus" character

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2008-03-18  Rafael Laboissiere  <rafael@debian.org>
+
+	* interpreter/octave.1, interpreter/mkoctfile.1:
+	Escape "-" signs that mean the "minus" character.
+
 2007-12-21  John W. Eaton  <jwe@octave.org>
 
 	Version 3.0.0 released.
 
 2007-12-13  John W. Eaton  <jwe@octave.org>
 
 	* Makefile.in (conf.texi): Also depend on $(top_srcdir)/src/version.h.
 
diff --git a/doc/interpreter/mkoctfile.1 b/doc/interpreter/mkoctfile.1
--- a/doc/interpreter/mkoctfile.1
+++ b/doc/interpreter/mkoctfile.1
@@ -16,21 +16,21 @@
 .\" You should have received a copy of the GNU General Public License
 .\" along with Octave; see the file COPYING.  If not, see
 .\" <http://www.gnu.org/licenses/>.
 .\"
 .\" This page was contributed by Dirk Eddelbuettel <edd@debian.org>
 .\"
 .TH MKOCTFILE 1 "1 November 2002" "GNU Octave"
 .SH NAME
-mkoctfile - Compile dynamic-load modules for GNU Octave
+mkoctfile \- Compile dynamic-load modules for GNU Octave
 .SH SYNOPSIS
-.BR mkoctfile\  [-IDIR]\ [-DDEF]\ [-lLIB]\ [-LDIR]\ [-M|--depend]\ 
-[-c]\ [-o FILE|--output FILE]\ [-p VAR|--print VAR]\ [-s|\--strip]\ 
-[-v|--verbose]\ [-h|-?|--help]\ \fIfile\fP .\|.\|.  
+.BR mkoctfile\  [\-IDIR]\ [\-DDEF]\ [\-lLIB]\ [\-LDIR]\ [\-M|\-\-depend]\ 
+[\-c]\ [\-o FILE|\-\-output FILE]\ [\-p VAR|\-\-print VAR]\ [\-s|\-\-strip]\ 
+[\-v|\-\-verbose]\ [\-h|\-?|\-\-help]\ \fIfile\fP .\|.\|.  
 .SH DESCRIPTION
 .PP
 \fImkoctfile\fP is used to compile source C, C++ or Fortran source code in
 dynamically loadable 
 .I .oct
 file for 
 .BR octave (1).
 .SH OPTIONS
@@ -50,17 +50,17 @@ Add library directory DIR to link comman
 .TP 8
 .B \-M|\-\-depend
 Generate dependency files (.d) for C and C++ source files.
 .TP 8
 .B \-c
 Compile but do not link.
 .TP 8
 .B \-o FILE|\-\-output FILE
-Output file name; default extension is .oct (or .mex if --mex is
+Output file name; default extension is .oct (or .mex if \-\-mex is
 specified) unless linking a stand-alone executable.
 .TP
 .B \-p VAR|\-\-print VAR
 Print configuration variable VAR.  Recognized variables are:
 .RS
     ALL_CFLAGS                FFTW_LIBS     
     ALL_CXXFLAGS              FLIBS       
     ALL_FFLAGS                FPICFLAG      
diff --git a/doc/interpreter/octave.1 b/doc/interpreter/octave.1
--- a/doc/interpreter/octave.1
+++ b/doc/interpreter/octave.1
@@ -24,33 +24,33 @@ octave \- A high-level interactive langu
 .nf
 octave [options]
 .fi
 .SH OPTIONS
 The complete set of command-line options for octave is available by
 running the command
 .nf
 
-    octave --help
+    octave \-\-help
 
 .fi
 .SH DESCRIPTION
 Octave is a high-level language, primarily intended for numerical
 computations.  It provides a convenient command line interface for
 solving linear and nonlinear problems numerically.
 .SH DOCUMENTATION
 The primary documentation for Octave is written using Texinfo, the GNU
 documentation system, which allows the same source files to be used to
 produce on-line and printed versions of the manual.
 .PP
 You can read the on-line copy of the Octave documentation by issuing
 the command
 .nf
 
-    octave:13> help -i
+    octave:13> help \-i
 
 .fi
 while running Octave interactively, by using the GNU Emacs info mode,
 or by running standalone programs like info or xinfo.
 .SH BUGS
 The best way to submit a bug report for Octave is to use the command
 .nf
 
